README.md

```diff
@@ -209,171 +208,15 @@ Defaults for AEMM configuration are sourced throughout code. Examples below:
-  }
-}
-```
-
-Use the `-c` flag to consume the configuration file and the `-s` flag to save an output of the configurations used by AEMM after precedence has been applied:
-
-```
-$ amazon-ec2-metadata-mock -c path/to/config-overrides.json -s
-Successfully saved final configuration to local file  /path/to/home/.amazon-ec2-metadata-mock/.aemm-config-used.json
-
-
-$ cat $HOME/.amazon-ec2-metadata-mock/.aemm-config-used.json
-(truncated for readability)
-
-{
-  "config-file": "path/to/config-overrides.json",
-  "metadata": {
-    "paths": {
-      "ipv4-associations": "/latest/meta-data/network/interfaces/macs/0e:49:61:0f:c3:77/ipv4-associations/192.0.2.54"
-    },
-    "values": {
-      "mac": "0e:49:61:0f:c3:77",
-      "public-ipv4": "54.92.157.77"
-    }
-  },
-  "mock-delay-sec": 12,
-  "save-config-to-file": true,
-  "server": {
-    "hostname": "localhost",
-    "port": "1338"
-  },
-  "spot-itn": {
-    "instance-action": "stop",
-    "time": "2020-01-07T01:03:47Z"
-  }
-}
-
-```
+## Overrides
+AEMM supports configuration from various sources including: cli flags, env variables, and config files. Details regarding
+configuration steps, behavior, and precedence are outlined [here](https://github.com/aws/amazon-ec2-metadata-mock/blob/master/docs/configuration.md).
 
 # Usage
-AEMM is primarily used as a developer tool to help test behavior related to Metadata Service. Popular use cases include: emulating spot instance interrupts after a designated delay, mocking scheduled maintenance events, and testing IMDSv1 to IMDSv2 migrations.
+AEMM is primarily used as a developer tool to help test behavior related to Metadata Service. Popular use cases include: emulating spot instance interrupts after a designated delay, mocking scheduled maintenance events, IMDSv2 migrations, 
+and requesting static metadata. This section outlines the common use cases of AEMM; advanced usage and behavior are documented [here](https://github.com/aws/amazon-ec2-metadata-mock/blob/master/docs/usage.md).
 
-## Commands
-AEMM's supported commands (`spotitn`, `scheduledevents`) are viewed using `--help`:
-```
-$ amazon-ec2-metadata-mock --help
-
-...
-Available Commands:
-  help            Help about any command
-  scheduledevents Mock EC2 Scheduled Events
-  spotitn         Mock EC2 Spot interruption notice
-
-...
-```
-commands are designed as follows:
-* Run independently from other commands
-  * i.e. when AEMM is started with `scheduledevents` subcommand, `spotitn` routes will **NOT** be available and vice-versa 
-* Local flag availability so that commands can be configured directly via CLI parameters
-    * With validation checks
-* Contain additional `--help` documentation
-* Default values sourced from code
-
-Metadata categories that are always available, irrespective of the CLI command run are referred to as **static metadata**. More details on static metadata can be found [here](#static-metadata)
-
-### Spot Interruption
+### Spot Interruptions
```

Comment:
You probably meant to keep `## Commands` here? 
[nit] `Interruption` (singular) here and in `usage.md`?

Reply:
- I can remove the 's' from interruptions in both docs
- I purposefully moved 'Commands' out of here because that seemed to be more of an implementation detail than a usability one. Given one of the big requests from AEMM was Spot ITN notices, I wanted to make those instructions as straightforward as possible without requiring background info.

What do you think?

Reply:
gotcha.. Then, may be you want a H2 header markdown to match with IMDS versions and static metadata?

Reply:
Bout it; I'll make the updates